helm/robusta/templates/runner.yaml

```diff
@@ -173,6 +173,7 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
     app: {{ .Release.Name }}-runner
+    target: robusta-runner
```

Comment:
<img width="1630" alt="image" src="https://github.com/robusta-dev/robusta/assets/20162469/d3127e4d-7d15-4854-b96a-bb7c9d5e3c87">

this adds the target label to metrics, other solution will be to name metrics with robusta prefix